### DISCUSSION TITLE
Mistake in nuclei-templates/vulnerabilities/generic/generic-blind-xxe.yaml ?

### DISCUSSION THREAD

**Discussion by vblonsky** [projectdiscovery/nuclei-templates#2508] [Q&A] [answered]
Why there is 2 empty lines between POST headers and body in nuclei-templates/vulnerabilities/generic/generic-blind-xxe.yaml? 
Shouldn't it be only one empty line?

**Comment by ehsandeep** [comment, marked as answer]
Thank you @vblonsky, this is now fixed at https://github.com/projectdiscovery/nuclei-templates/pull/2531
